core/numbers/numbers.py

```diff
@@ -177,6 +177,7 @@ def split_list(value, l: list) -> Iterator:
 number_small_map = {n: i for i, n in enumerate(number_small_list)}
 
 mod.list("number_small", desc="List of small numbers")
+mod.tag("disable_unprefixed_numbers", desc="Require prefix when saying a number")
```

Comment:
I might suggest just calling this "prefixed_numbers"

Reply:
See other comment about why it's this way.

Reply:
Could call it "numbers must be prefixed" if the double negative is too confusing?

Reply:
Ok, I'll update it to be non-double negative tag then. user.prefix_numbers or user.numbers_must_be_prefixed? I think I prefer the latter because it's a bit more self explanatory, but good with whatever others prefer.